### PR TITLE
Add explanation for table names.

### PR DESCRIPTION
**Pull request by bazile-clyde**
It took me a minute to figure out why the table names were what they were. I'm not sure if this is documented somewhere else but I thought this might be useful for others.

### CODE REVIEW

**Review by rchateauneu — APPROVED** [review]
ok.

**Comment by bazile-clyde** [comment]
> ok.

@rchateauneu Thanks! I still see `At least 1 approving review is required by reviewers with write access.` and `Merging can be performed automatically with 1 approving review.` So I can't merge yet. I guess I have to wait for an additional reviewer?

**Comment by rchateauneu** [comment]
> 
> 
> > ok.
> 
> @rchateauneu Thanks! I still see `At least 1 approving review is required by reviewers with write access.` and `Merging can be performed automatically with 1 approving review.` So I can't merge yet. I guess I have to wait for an additional reviewer?

Yes, because I do not have write access.

**Review by mwatts15 — APPROVED** [review]

**Pull request merged**
